src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -12,11 +13,19 @@ import { useServerData } from '../../../lib/providers';
 import { currencyFormat } from '../../../lib/hooks/currency'; // Remove .00 cents from formated min and max
 
 export const ShortTerm = ({
-    content: { instructions, linkToProductList, estimatedInstallments, disclosure, donutTimestamps, learnMoreLink },
-    productMeta: { qualifying, periodicPayment },
+    content: {
+        instructions,
+        linkToProductList,
+        estimatedInstallments,
+        preapproval: { preapprovalDisclaimerHeadline, preapprovalDisclaimerBody },
```

Comment:
```suggestion
        preapproval,
```
Let's update this to just be `preapproval` since it can be potentially undefined.